### PR TITLE
increase max line length to 49900 for scalyr agent

### PR DESCRIPTION
**Pull request by pheanex**
default (before) was [9900](https://www.scalyr.com/help/scalyr-agent-env-aware)
"read_page_size" also increased as recommended by scalyr-support

### CODE REVIEW

**Comment by pheanex** [comment]
:+1:

**Comment by pheanex** [comment]
:+1:

File: kube_log_watcher/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '0.34'
+__version__ = '0.36'
```
Comment (astarnell) [inline kube_log_watcher/__init__.py]: Why `0.36`? Shouldn't `0.35` be the next version?

Comment (pheanex) [reply kube_log_watcher/__init__.py]: Because I've forgot to bump the version the last time (removing the empty newlines, see changelog))

**Comment by pheanex** [comment]
:+1:

**Comment by astarnell** [comment]
:+1:

**Pull request merged**